src/en/charms-relations.md

```diff
@@ -49,19 +50,27 @@ For example, in the case of specifying a database for the Mediawiki charm.
 
 ```bash
 juju add-relation mediawiki mysql
-error: ambiguous relation: "mediawiki mysql" could refer to 
+error: ambiguous relation: "mediawiki mysql" could refer to
   "mediawiki:db mysql:db"; "mediawiki:slave mysql:db"
 ```
 
-The solution is to specify the nature of the relation using the
-relation interface identifier. In this case, we want MySQL to provide the 
-backend database for mediawiki ('db' relation), so this is what we need to 
-enter:
+The solution is to be explicit when referring to an *endpoint*, where the
+latter has a format of `<application>:<application endpoint>`. In this case, it
+is 'db' for both applications. However, it is not necessary to specify the
+mysql endpoint because only the mediawiki endpoint is ambiguous (according to
+the error message). Therefore, the command becomes:
 
 ```bash
-juju add-relation mediawiki:db mysql
+juju add-relation mysql mediawiki:db
```

Comment:
why is the order reversed? I mean, it shouldn't matter, but we have always used <from> <to> order in the examples

Reply:
From the thing doing the giving, to the consumer. wordpress consumes mysql, as does mediawiki

Reply:
So doesn't my new order follow your policy? You said the giver comes first. I've changed it so that mysql, the giver (doing the giving), comes first.

Reply:
Yes, sorry it does now